ppl/src/main/java/com/amazon/opendistroforelasticsearch/sql/ppl/plans/expression/Function.java

```diff
@@ -0,0 +1,52 @@
+/*
+ *   Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+package com.amazon.opendistroforelasticsearch.sql.ppl.plans.expression;
+
+import com.amazon.opendistroforelasticsearch.sql.ppl.node.AbstractNodeVisitor;
+import com.amazon.opendistroforelasticsearch.sql.ppl.node.NodeVisitor;
+import java.util.Arrays;
+import java.util.List;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.ToString;
+
+/**
+ * Expression node of scalar function
+ * Params include function name (@funcName) and function arguments (@funcArgs)
+ */
+@Getter
+@ToString
+@EqualsAndHashCode
+@RequiredArgsConstructor
+public class Function extends Expression {
+    private final Expression funcName;
```

Comment:
I think funcName is just Name instead of Expression.

Reply:
Ok yes that sounds better